gadfly/sentence_summarizer.py

```diff
@@ -58,3 +59,78 @@ def summarize(self, sents, n):
     def _rank(self, ranking, n):
         """ return the first n sentences with highest ranking """
         return nlargest(n, ranking, key=ranking.get)
+
+
+class TF_IDFSummarizer:
+    def __init__(self):
+        pass
+
+    def get_tokens_and_freqs(self, sents, style):
+        """Get a list of all tokens in sents and a Counter of them"""
+        if style == 'standard':

```

Comment:
Can you change this to an ENUM? See the pattern used in gap_fill_generator  in development branch.